src/pages/_document.tsx

```diff
@@ -1,36 +1,47 @@
 import NextDocument, { Html, Main, NextScript, Head } from 'next/document'
```

Comment:
do we need to import NextDocument still?

Reply:
We do not, will remove